internal/pkg/aws/cloudformation/testdata/parse/lb-web-svc.yaml

```diff
@@ -140,6 +140,10 @@ Resources:
       DeploymentConfiguration:
         MinimumHealthyPercent: 100
         MaximumPercent: 200
+        Alarms:
+          AlarmNames: []
+          Enable: false
+          Rollback: true
```

Comment:
Note that all three are required field, and `UpdateService` won't pick up any changes if they are not specified.